### ISSUE TITLE
Textslider training help with pretrained LORA

### ISSUE THREAD

**Issue by aretius** [rohitgandikota/sliders#113]
Hello
I wanted to train a text slider and i have a SDXL model with a pretrained LORA. I want the text slider to use this model and LORA for the slider training - basically create embeddings/images based on my custom model. 
Is that possible? What part of code will I have to make edit on?

**Comment by rohitgandikota** [comment]
Yes it is possible. We don't have code for this specific situation. One way you can do it is:

1. Merge your LoRA to the original model (you can add this code after you load the model in the slider training code)
2. Train you slider as per usual
3. During inference you can load both you custom LoRA and slider at the same time

**Comment by rohitgandikota** [comment]
Hey please refer to our latest release: https://github.com/rohitgandikota/sliders/blob/main/concept_sliders_diffusers.ipynb

this might help with the issue
